### DISCUSSION TITLE
Training/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#82] [Show and tell]
# Training/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://hello-ctf.com/Training/

**Comment by slagman34** [comment]
来题来题

**Reply by XDWkk** [reply]
题目捏

**Comment by ProbiusOfficial** [comment]
肚子里（雾



---Original---
From: ***@***.***&gt;
Date: Tue, Jul 23, 2024 20:39 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [ProbiusOfficial/Hello-CTF] Training/ (Discussion #82)




 
题目捏

**Comment by XINDENG123456** [comment]
嘻嘻
